code/game/machinery/doors/unpowered.dm

```diff
@@ -18,3 +18,6 @@
 /obj/machinery/door/unpowered/shuttle
 	icon = 'icons/turf/shuttle.dmi'
 	icon_state = "door1"
+	SdoorOpen = 'sound/machines/airlock_open.ogg'
+	SdoorClose = 'sound/machines/airlock_close.ogg'
+	autoclose = TRUE
```

Comment:
This isn't the way to do it, this is literally a copy-pasted that's on a different subtype. It's pretty gross.

Either refactor or change the type of doors on the shuttle (not recommended).

Reply:
couldn't i define the sounds for the airlocks all the way down at door and then just use it there instead?